src/js/UploadButton.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import classNames from 'classnames';
+import $ from 'jquery';
+
+import css from '../stylesheets/button.css';
+
+export default class UploadButton extends React.Component {
+   /**
+      * Base Button
+      * @param  {Object} props
+   */
+   constructor(props) {
+      super(props);
+      this.state = {
+         buttonLabel: ''
+      };
+      this.handleChange = this.handleChange.bind(this);
+   }
+
+   /**
+      * Handle the button text label after component rendered
+   */
+   componentDidMount () {
+      let $this = $(this.myTextInput),
+         label = this.props.label;
+      $this.attr('data-content', label);
+   }
+
+   /**
+      * Handle the button state
+      * @param  {Event]} event
+   */
+   handleChange (e) {
+      this.setState({
+         buttonLabel: e.target.value.split(/(\\|\/)/g).pop()
+      });
+      if (this.props.onChange) {
+         this.props.onChange(e);
+      }
+   }
+
+   render() {
+      // let text = this.props.store.submissionInProgress ? 'Processing...' : 'Submit';
+      let { color } = this.props;
+      let btnClasses = classNames(
+         css.btn,
+         css.raised,
+         css.flexItem,
+         css.input,
+         css[color]
+      );
+      // Construct props
+      // let props = {
```

Comment:
Why are these commented out? Either leave a comment explaining why something is commented out or don't commit the comments please.